### PR TITLE
Initial implementation of risk calculation in EN v2

### PR DESCRIPTION
**Pull request by jonasofrich**

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19/ExposureNotifications/ExposureNotificationHandler.cs [hunk 1]
```diff
@@ -175,10 +175,19 @@ public Task<DailySummaryConfiguration> GetDailySummaryConfigurationAsync()
...
-            throw new NotImplementedException();
+            bool shouldSendMessage = false;
+            foreach (DailySummary dailySummary in summaries)
+            {
+                if (ExposureDetectedHelper.RiskInDailySummaryAboveThreshold(dailySummary))
+                {
+                    shouldSendMessage = true;
+                    break;
+                }
+            }
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19/ExposureNotifications/ExposureNotificationHandler.cs:189]: That's fine for now I guess, but I think we can do even more advanced calculation/scoring in the end :) We will see :)

File: NDB.Covid19/NDB.Covid19/ExposureNotifications/Helpers/ExposureDetected/ExposureDetectedHelper.cs [hunk 2]
```diff
@@ -14,6 +14,13 @@ public abstract class ExposureDetectedHelper
         static string _logPrefix = $"{nameof(ExposureDetectedHelper)}";
         private static SecureStorageService _secureStorageService => ServiceLocator.Current.GetInstance<SecureStorageService>();
 
+        public static bool RiskInDailySummaryAboveThreshold(DailySummary dailySummary)
+        {
+            double thresholdInSeconds = LocalPreferencesHelper.ExposureTimeThreshold * 60;
+            if (dailySummary.Summary.WeightedDurationSum >= thresholdInSeconds) return true;
+            return false;
+        }
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19/ExposureNotifications/Helpers/ExposureDetected/ExposureDetectedHelper.cs:22]: We want unit tests for this logic here

**Review by pavshr — CHANGES_REQUESTED** [review]

File: NDB.Covid19/NDB.Covid19/ExposureNotifications/Helpers/ExposureDetected/ExposureDetectedHelper.cs (hunk 2 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19/ExposureNotifications/Helpers/ExposureDetected/ExposureDetectedHelper.cs:22]: Implemented

File: NDB.Covid19/NDB.Covid19/ExposureNotifications/ExposureNotificationHandler.cs (hunk 1 above)
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19/ExposureNotifications/ExposureNotificationHandler.cs:192]: @jonasofrich 
Infectiousness.None and ReportType.Unknown and ReportType.Revoked should be removed from configuration as EN throws exceptions if we use them

Comment (pavshr) [reply NDB.Covid19/NDB.Covid19/ExposureNotifications/ExposureNotificationHandler.cs:192]: @nc-kano I am making a PR with that

**Review by pavshr — APPROVED** [review]

**Pull request merged**